wheel/src/api.rs

```diff
@@ -77,6 +80,15 @@ pub fn compute_merkle_set_root<'p>(
     Ok(PyBytes::new(py, &compute_merkle_root_impl(&mut buffer)))
 }
 
+#[pyfunction]
+pub fn deserialize_proof(proof: Py<PyBytes>) -> PyResult<MerkleSet> {
+    let bytes = Python::with_gil(|py| proof.as_bytes(py));
+    match deserialise_proof(bytes) {
```

Comment:
You can use `.map_err(|_| exceptions::PyValueError::new_err("Error deserialising proof."))` instead of this `match`